From 2b2853035ef3a13559d296ea5e8625d55e3369bd Mon Sep 17 00:00:00 2001
From: Naga Sureshkumar Relli <naga.sureshkumar.relli@xilinx.com>
Date: Fri, 16 Jun 2017 19:02:46 +0530
Subject: [PATCH 137/532] can: xilinx: Fix sparse warning in driver

commit 585a996977be7c3c5ec6cc5c5d02bd455e58a8de from
https://github.com/Xilinx/linux-xlnx.git

Fixes the below warning

drivers/net/can/xilinx_can.c:548:5: warning: symbol 'xcan_get_freebuffer'
was not declared. Should it be static?

Signed-off-by: Naga Sureshkumar Relli <nagasure@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/net/can/xilinx_can.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/can/xilinx_can.c b/drivers/net/can/xilinx_can.c
index bf39b52..45a1ca3 100644
--- a/drivers/net/can/xilinx_can.c
+++ b/drivers/net/can/xilinx_can.c
@@ -545,7 +545,7 @@ static int xcan_do_set_mode(struct net_device *ndev, enum can_mode mode)
  *
  * Return: Free Buffer on success and -1 if no buffer available
  */
-int xcan_get_freebuffer(struct xcan_priv *priv)
+static int xcan_get_freebuffer(struct xcan_priv *priv)
 {
 	u32 bufindex = 0, trrregval = 0;
 
-- 
2.7.4

